site/docs/v1/tech/apis/_entity-type-response-body-base.adoc

```diff
@@ -0,0 +1,63 @@
+[api]
+ifeval::["{base_field_name}" != "entityType"]
+[field]#{base_field_name}# [type]#[Array]#::
+The list of EntityType objects.
+endif::[]
+
+[field]#{base_field_name}.data# [type]#[Object]# [optional]#Optional#::
```

Comment:
I believe we omit Optional, Required, and "defaults to" in response body documentation.

Reply:
100%, thanks!